app/controllers/api/configuration_script_sources_controller.rb

```diff
@@ -1,4 +1,26 @@
 module Api
   class ConfigurationScriptSourcesController < BaseController
+    def create_resource(_type, _id, data)
+      attrs = validate_attrs(data)
+      type = ConfigurationScriptSource.class_from_request_data(attrs)
+      raise 'type not currently supported' unless type.respond_to?(:create_in_provider_queue)
```

Comment:
once the model supports other than ExtManagementSystem, we can provide an different variant of parse_id that supports a list of collections, where it returns the collection and id. This way we know for sure what href we are parsing (and the correct collection accessed in the following resource_search).